src/Extension/ExtensionInterface.php

```diff
@@ -67,8 +67,8 @@ public function getFunctions();
      * @return array<array> First array of unary operators, second array of binary operators
      *
      * @psalm-return array{
-     *     array<string, array{precedence: int, class: class-string<AbstractExpression>}>,
-     *     array<string, array{precedence: int, class?: class-string<AbstractExpression>, associativity: ExpressionParser::OPERATOR_*}>
+     *     array<string, array{precedence: int, future_precedence: int, package: string, version: string, class: class-string<AbstractExpression>}>,
+     *     array<string, array{precedence: int, future_precedence: int, package: string, version: string, class?: class-string<AbstractExpression>, associativity: ExpressionParser::OPERATOR_*}>
```

Comment:
the keys need to be optional:

```suggestion
     *     array<string, array{precedence: int, future_precedence?: int, package?: string, version?: string, class?: class-string<AbstractExpression>, associativity: ExpressionParser::OPERATOR_*}>
```

Reply:
Done